internal/utils/templates/initial_schemas/14.sql

```diff
@@ -103,6 +103,20 @@ CREATE EXTENSION IF NOT EXISTS pgjwt WITH SCHEMA extensions;
 COMMENT ON EXTENSION pgjwt IS 'JSON Web Token API for Postgresql';
 
 
+--
+-- Name: pgtap; Type: EXTENSION; Schema: -; Owner: -
+--
+
+CREATE EXTENSION IF NOT EXISTS pgtap WITH SCHEMA extensions;
+
+
+--
+-- Name: EXTENSION pgtap; Type: COMMENT; Schema: -; Owner:
+--
+
+COMMENT ON EXTENSION pgtap IS 'Unit testing for PostgreSQL';
+
+
```

Comment:
No need to install pgtap here - pg_prove installs it automatically. pgtap is also not installed by default on the hosted platform